db/migrate/20181010192630_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre
+      t.string :autor
+      t.string :image
+      t.string :title
+      t.string :editor
+      t.string :year
+
```

Comment:
add description

Reply:
@mtejedorwolox are we going to use that attribute as string or text type?